kernel/api/src/main/java/org/sakaiproject/event/api/TincanapiEvent.java

```diff
@@ -0,0 +1,34 @@
+package org.sakaiproject.event.api;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Entity
+@Table( name = "TINCANAPI_EVENT")
+@NamedQuery( name = "getAllTincanapiEvent",
+			 query = "from TincanapiEvent")
+
+@Data
+public class TincanapiEvent {

```

Comment:
Is a table really needed, could a queue be sufficient?

Reply:
In this case, the Table is used to save the different type of event, not the event itself. It's an easy way to add new ones.
The XApi event is not stored, it's just sent.
Thanks